include/deal.II/base/template_constraints.h

```diff
@@ -803,40 +804,40 @@ namespace concepts
 
 #  ifdef DEAL_II_WITH_PETSC
     template <>
-    constexpr bool is_dealii_vector_type<dealii::PETScWrappers::Vector> = true;
+    inline constexpr bool is_dealii_vector_type<dealii::PETScWrappers::Vector> =
+      true;
 
     template <>
-    constexpr bool is_dealii_vector_type<dealii::PETScWrappers::BlockVector> =
-      true;
+    inline constexpr bool
+      is_dealii_vector_type<dealii::PETScWrappers::BlockVector> = true;
 
     template <>
-    constexpr bool is_dealii_vector_type<dealii::PETScWrappers::MPI::Vector> =
-      true;
+    inline constexpr bool
+      is_dealii_vector_type<dealii::PETScWrappers::MPI::Vector> = true;
 
     template <>
-    constexpr bool
+    inline constexpr bool
       is_dealii_vector_type<dealii::PETScWrappers::MPI::BlockVector> = true;
 #  endif
 
 #  ifdef DEAL_II_WITH_TRILINOS
     template <>
-    constexpr bool
+    inline constexpr bool
       is_dealii_vector_type<dealii::TrilinosWrappers::MPI::Vector> = true;
 
     template <>
-    constexpr bool
+    inline constexpr bool
       is_dealii_vector_type<dealii::TrilinosWrappers::MPI::BlockVector> = true;
 
     template <>
-    constexpr bool
+    inline constexpr bool
       is_dealii_vector_type<dealii::LinearAlgebra::EpetraWrappers::Vector> =
         true;
 
 #    ifdef DEAL_II_TRILINOS_WITH_TPETRA
-    template <>
-    constexpr bool
-      is_dealii_vector_type<dealii::LinearAlgebra::TpetraWrappers::Vector> =
-        true;
+    template <typename Number>
+    constexpr bool is_dealii_vector_type<
```

Comment:
Do you want to mark this one as `inline` as well?

Reply:
Sure, fixed. Let's be consistent. 🙂